phase3/calico/calico-node-daemonset.json

```diff
@@ -0,0 +1,179 @@
+{
+    "kind": "DaemonSet", 
+    "spec": {
+        "template": {
+            "spec": {
+                "containers": [
+                    {
+                        "securityContext": {
+                            "privileged": true
+                        }, 
+                        "name": "calico-node", 
+                        "env": [
+                            {
+                                "valueFrom": {
+                                    "configMapKeyRef": {
+                                        "name": "calico-config", 
+                                        "key": "etcd_endpoints"
+                                    }
+                                }, 
+                                "name": "ETCD_ENDPOINTS"
+                            }, 
+                            {
+                                "valueFrom": {
+                                    "configMapKeyRef": {
+                                        "name": "calico-config", 
+                                        "key": "calico_backend"
+                                    }
+                                }, 
+                                "name": "CALICO_NETWORKING_BACKEND"
+                            }, 
+                            {
+                                "name": "CALICO_DISABLE_FILE_LOGGING", 
+                                "value": "true"
+                            }, 
+                            {
+                                "name": "FELIX_DEFAULTENDPOINTTOHOSTACTION", 
+                                "value": "ACCEPT"
+                            }, 
+                            {
+                                "name": "CALICO_IPV4POOL_CIDR", 
+                                "value": "192.168.0.0/16"
```

Comment:
I don't think this is the cause of the error, but this needs to be updated to match the cluster-cidr: 10.244.0.0/16